client/src/document/spec-table.tsx

```diff
@@ -0,0 +1,30 @@
+import { DisplayH2, DisplayH3 } from "./ingredients/utils";
+
+export function SpecificationTable({
+  id,
+  title,
+  isH3,
+  specURLs,
+}: {
+  id: string;
+  title: string;
+  isH3: boolean;
+  specURLs: string[];
+}) {
+  return (
+    <>
+      {title && !isH3 && <DisplayH2 id={id} title={title} />}
+      {title && isH3 && <DisplayH3 id={id} title={title} />}
+      <table className="standard-table">
+        <thead>
+          <tr>
+            <th scope="col">Specification</th>
+          </tr>
+        </thead>
+        <tr>
+          <td>{specURLs}</td>
```

Comment:
Was this what you were referring to fully fleshing out in the PR comment?

Reply:
Yeah, this needs work. Just dumping in URLs is not it :) Will work more on this next week.